auth/src/main/java/com/firebase/ui/auth/AuthUI.java

```diff
@@ -470,13 +472,23 @@ public SignInIntentBuilder setProviders(@NonNull String... providers) {
         /**
          * Enables or disables the use of Smart Lock for Passwords in the sign in flow.
          * 
-         * <p>SmartLock is enabled by default
+         * <p>SmartLock is enabled by default.
          */
         public SignInIntentBuilder setIsSmartLockEnabled(boolean enabled) {
             mIsSmartLockEnabled = enabled;
             return this;
         }
 
+        /**
+         * Links the current user to an account created in the sign-in flow.
+         *
+         * <p>Linking is disabled by default.
+         */
+        public SignInIntentBuilder setShouldLinkAccounts(boolean shouldLinkAccounts) {
```

Comment:
Thoughts on the new name? `setShouldLinkAccounts` vs `setShouldLinkUser`. I prefer the first but it's up to debate.